### PR TITLE
refactor: using definite patterns to simplify keybindings

### PR DESCRIPTION
**Pull request by sxyazi**

### CODE REVIEW

**Comment by Brixy** [comment]
Thanks, @sxyazi, for all your work and excellent additions to yazi.

I noticed the following issue with this pull request:

When you do a `yp`, files vanish. I don’t know what happens technically, but the files are not moved into the trash folder.

File: yazi-core/src/manager/commands/paste.rs [hunk 1]
```diff
@@ -18,10 +18,11 @@ impl Manager {
 		let opt = opt.into() as Opt;
 
 		let dest = self.cwd();
-		if self.yanked.cut {
-			tasks.file_cut(&self.yanked, dest, opt.force);
-		} else {
+		if !self.yanked.cut {
 			tasks.file_copy(&self.yanked, dest, opt.force, opt.follow);
```
Comment (Rolv-Apneseth) [inline yazi-core/src/manager/commands/paste.rs:22]: Is the bug possibly cause of a missing `return` here?

Comment (sxyazi) [reply yazi-core/src/manager/commands/paste.rs:22]: oops I missed it in reviewing, let me fix it

Comment (sxyazi) [reply yazi-core/src/manager/commands/paste.rs:22]: fixed in https://github.com/sxyazi/yazi/commit/2f784e7ae71c07e02f318f9a6d3c45c586253e59

Comment (Rolv-Apneseth) [reply yazi-core/src/manager/commands/paste.rs:22]: Nice

**Comment by Rolv-Apneseth** [comment]
Thanks for reporting @Brixy

**Pull request merged**